src/sagemaker_pytorch_serving_container/torchserve.py

```diff
@@ -92,7 +88,7 @@ def start_torchserve(handler_service=DEFAULT_HANDLER_SERVICE):
         "--log-config",
         DEFAULT_TS_LOG_FILE,
         "--models",
-        "model.mar"
+        DEFAULT_TS_MODEL_NAME=environment.model_dir
```

Comment:
Does this need to be `DEFAULT_TS_MODEL_NAME + "=" + environment.model_dir`?

Reply:
addressed